### PR TITLE
Create how-to-deploy-on-AWS.md

### PR DESCRIPTION
**Pull request by pranaygon**
Hello Team,

It would be great opportunity for me to contribute a simple document about "How to deploy Verdaccio Private Registry on AWS". I have already written this post in my blog (https://www.howtoinmagento.com/2019/11/free-verdaccio-private-registry-setup.html)

This question is quite popular (https://github.com/verdaccio/website/issues/6) and would be a great doc page to help other.

It would be appreciated if you can review my PR and provide your feedback.

The best place for this doc will be under guides section if everything looks good!

Thank You!

Pranay Gondane (pranaydac08@gmail.com)

### CODE REVIEW

File: docs/how-to-deploy-on-AWS.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---
+id: aws
+title: "Amazon Web Services"
+---
+
+This document describes simple steps to setup Verdaccio private registry on Amazon Web Services platform using EC2 service. Considering you already created EC2 Amazon Linux instance, if not then please check this tutorial [AWS EC2 Setup](https://tinyurl.com/y2ozksyc).
```
Comment (juanpicado) [inline docs/how-to-deploy-on-AWS.md:6]: I'd rahter use your original URL, `https://tinyurl.com` might be unvalid eventually.

**Review by juanpicado — DISMISSED** [review]
Brilliant 💟

File: docs/how-to-deploy-on-AWS.md (hunk 1 above)
Comment (pranaygon) [reply docs/how-to-deploy-on-AWS.md:6]: sure

Comment (pranaygon) [reply docs/how-to-deploy-on-AWS.md:6]: Let me update original URL

Comment (pranaygon) [reply docs/how-to-deploy-on-AWS.md]: Done. Please review again and approve.

**Review by juanpicado — APPROVED** [review]
Perfect !

**Comment by juanpicado** [comment]
Thanks so much !

**Comment by pranaygon** [comment]
Thank you very much for you idea. Much appreciated!

**Comment by pranaygon** [comment]
Please let me know once it live on production site. Thanks!

**Pull request merged**